### PR TITLE
fix(tabs): added tabs overflow documentation

### PR DESCRIPTION
**Pull request by Rajdeepc**
## Description

Due to missing access specifiers in property and slot references in the component comments, these APIs were not getting populated in the API section of the `sp-tabs-overflow` component documentation

## Related issue(s)

-  #4146 

## How has this been tested?

-   [x] _Test case 1_
    1. Go [here](https://bug-taboverflow-api-docs--spectrum-web-components.netlify.app/components/tabs-overflow/api/)

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: packages/tabs/src/TabsOverflow.ts [hunk 1]
```diff
@@ -39,14 +39,16 @@ interface TabsOverflowState {
 }
 /**
  * @element sp-tabs-overflow
+ * @slot - text label of the Tab
+ * @slot icon - The icon that appears on the left of the label
```
Comment (Westbrook) [inline packages/tabs/src/TabsOverflow.ts:43]: I'm not sure this is in the right file. Is this documentation for the `<sp-tab>` element or the `<sp-tabs-overflow>` element?

**Review by Westbrook — COMMENTED** [review]
Can you catch up to main?

File: packages/tabs/src/TabsOverflow.ts (hunk 1 above)
Comment (Rajdeepc) [reply packages/tabs/src/TabsOverflow.ts:43]: `<sp-tab-overflow>` also consider these two slots same as a `<sp-tab>` component. I can remove this from here if this make sense for the consumers.

Comment (Westbrook) [reply packages/tabs/src/TabsOverflow.ts:43]: I don't see these slots in this file. Can you help me understand why you think they belong here?

Comment (Rajdeepc) [reply packages/tabs/src/TabsOverflow.ts:43]: Oh yes! You are right!! slot icon and slot text are part of `Tab.ts`. Rebased now to main.

Comment (Westbrook) [reply packages/tabs/src/TabsOverflow.ts]: This no longer seems to have any changes in it. Were some missing in the rebase, or can we close this now?

Comment (Rajdeepc) [reply packages/tabs/src/TabsOverflow.ts]: Yes we can close this!!

**Pull request closed without merging**